apps sc: Add securityadmin job and alerting roles (#1044)
elastisys/compliantkubernetes-apps: WIP-CHANGELOG.md

- Add option to encrypt off-site buckets replicated with rclone sync
- Added metrics for field mappings and an alert that will throw an error if the fields get close to the max limit.
- Add support for automatic reloading of the security config for OpenSearch

>   - **Warning**: When this runs it will replace all created users, roles, and role mappings, so create a backup or be prepared to recreate the manually created users.

Reviewer: ```suggestion
  - **Warning**: When this runs all users, roles, and role mappings created via the API will be removed, so create a backup or be prepared to recreate the manually created users.
```
